<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogStatusInfo</class>
 <widget class="QDialog" name="DialogStatusInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>50</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Status Info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnUpdate">
         <property name="text">
          <string>VSTATUS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEsync">
         <property name="text">
          <string>ESYNC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt1Command"/>
       </item>
       <item>
        <widget class="QComboBox" name="cbAllDrivers">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
